backend/manager/modules/bll/src/main/java/org/ovirt/engine/core/bll/storage/backup/StartVmBackupCommand.java

```diff
@@ -590,22 +582,13 @@ private boolean isBackupContainsRawDisksOnly() {
                 .noneMatch(DiskImage::isQcowFormat);
     }
 
-    private VmBackupInfo performVmBackupOperation(VDSCommandType vdsCommandType) {
+    private VmBackupInfo performVmBackupOperation(VDSCommandType vdsCommandType,
+                                                  Map<Guid, ScratchDiskInfo> scratchDiskInfoMap) {
```

Comment:
this indentation looks a bit too-good for our formatter :) please make sure to apply the formatter

Reply:
I tried again to format this with my IntelliJ Idea IDE and it produced the same formatting :)
I don't think I somehow changed the defaults.
What formatting did you expect?

Reply:
on my IDE (Eclipse) that is set with [the engine-code-formatter](https://github.com/oVirt/ovirt-engine/blob/master/config/engine-code-format.xml), the second line begins right below "VmBackupInfo"

Reply:
OK, found the relevant settings in `IntelliJ Idea` IDE and fixed the indentation, though I liked the previous style more.
I was not aware there is a code formatter for Eclipse under `ovirt-engine` project, it could be nice to have the same for `IntelliJ Idea`. I know what will you suggest now :)

Reply:
hehe, no - I wouldn't suggest to replace the IDE :) InteliJ Idea used to have the ability to import the same code formatter - not sure if it still exists these days though